rsconf/package_data/rsconf/rsconf.sh

```diff
@@ -309,6 +312,19 @@ rsconf_install_symlink() {
     rsconf_service_file_changed "$new"
 }
 
+rsconf_is_perl_rpm_rollback() {
+    declare prev_version=$1
+    declare install_version=$2
+    if ! rsconf_perl_rpm_version prev_version "$prev_version"; then
+        return 1
+    fi
+    rsconf_perl_rpm_version install_version "$install_version"
```

Comment:
It would be an error to have the value of `install_version` to be "not installed". In this case, it should be:

```bash
if ! rsconf_perl_rpm_version ...; then
   install_err "<program error> install_version=$install_version"
fi
```

The problem is if `rsconf_perl_rpm_version` returns false (1), the program will just exist without really telling you anything. Unlike Python (or Perl, the way we program it), bash doesn't give stack traces.

Think of something appropriate for `<program error>`. It might just be program error.

Reply:
fd1bc24

Good catch. I couldn't think of anything better than `program error`.